.github/CODEOWNERS

```diff
@@ -351,7 +339,6 @@ app/policies/debt_policy.rb @department-of-veterans-affairs/vsa-debt-resolution
 app/policies/demographics_policy.rb@department-of-veterans-affairs/mobile-api-team
 app/policies/form1095_policy.rb @department-of-veterans-affairs/vfs-1095-b @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
 app/policies/hca_disability_rating_policy.rb @department-of-veterans-affairs/vfs-authenticated-experience-backend @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-app/policies/lighthouse_policy.rb @department-of-veterans-affairs/lighthouse-banana-peels
```

Comment:
I don't see any of these files within `app/policies` being replaced with any top-level ownership. If lighthouse-banana-peels doesn't own them, please replace it with `@department-of-veterans-affairs/backend-review-group`